sites/en/installfest/editors.step

```diff
@@ -1,12 +1,12 @@
 message <<-MARKDOWN
 
-There are a number of different editors designed for programming. You may already have a favorite editor; if so you can continue using that editor. If you have never used a programming editor, there are a number of good editors available, several of them free of charge. Most Railsbridge instructors use the Sublime editor, so you may want to install Sublime; if you end up asking for help during your class it may be easier for an instructor or TA to help you if you are using Sublime. Here is a list of some of the more widely used editors:
+There are a number of different editors designed for programming. You may already have a favorite editor; if so you can continue using that editor. If you have never used a programming editor, there are a number of good editors available, several of them free of charge. Most RailsBridge instructors use the Atom editor, so you may want to install Atom; if you end up asking for help during your class it may be easier for an instructor or TA to help you if you are using Atom. Here is a list of some of the more widely used editors:
 
+* <a href="https://atom.io/" target="_blank">Atom</a> is a free, open-source editor that can be customized with HTML, CSS, and JavaScript. A download is available for OS X 10.8+, Windows 7/8, and Ubuntu/RedHat linux.
```

Comment:
I think this is saying that Atom itself can be extended with those technologies, not that those are the languages you use with it. 

From the Atom website:

> Customization
>
> It's easy to customize and style Atom. You can tweak the look and feel of your UI with CSS/Less and add major features with HTML and JavaScript.

Maybe more clear would be "extended" rather than "customized"? Or maybe that detail is just unnecessary for someone downloading their first editor?